modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -557,24 +561,57 @@ public CompletableFuture<?> onUpdate(ConfigurationNotificationEvent<Distribution
 
             int oldScaleUp;
 
+            int oldScaleDown;
+
             // ctx.oldValue() could be null for the default zone on a first start.
             if (ctx.oldValue() == null) {
                 oldScaleUp = Integer.MAX_VALUE;
+
+                oldScaleDown = Integer.MAX_VALUE;
```

Comment:
Is it `Integer.MAX_VALUE` or `default value for ScaleUp/ScaleDown`? If second then need to create constants for ScaleUp/ScaleDown default values.

Reply:
fixed